src/environment/stella_environment.cpp

```diff
@@ -178,6 +178,12 @@ void StellaEnvironment::soft_reset(){
   m_state.incrementFrame();
 }
 
+/** Emulates a NOOP action for both players */
+void StellaEnvironment::wait()
```

Comment:
because the score is set to 9997 then 9998 then 9999 before going back to 0000.
Without the wait function, we would need to deal with this in the code receiving the score. I thought that would be simpler to deal with it in the ALE.